.github/workflows/php_ci.yml

```diff
@@ -1,11 +1,5 @@
 name: PHP CI
-on:
```

Comment:
Any specific reason this is being changed back?

Reply:
I changed this so that CI can be run for merge requests from any branch.

Reply:
We purposely limit to main per our security team's requirements.  As long as the PR is _to_ the main branch, CI should run on it just fine.  So this probably needs to be changed back.